dracut-module-setup.sh

```diff
@@ -564,13 +608,34 @@ kdump_install_resolv_conf() {
     #
     # [1] https://bugzilla.gnome.org/show_bug.cgi?id=690404
     # [2] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/manually-configuring-the-etc-resolv-conf-file_configuring-and-managing-networking
-    systemctl -q is-enabled systemd-resolved 2> /dev/null && return 0
     inst "$_resolv_conf"
     if NetworkManager --print-config | grep -qs "^dns=none"; then
         printf "[main]\ndns=none\n" > "${initdir}/${_nm_conf_dir}"/90-dns-none.conf
     fi
 }
 
+kdump_install_ovs_deps() {
+    [[ $has_ovs_bridge == yes ]] || return 0
+    inst_multiple -o $(rpm -ql NetworkManager-ovs) $(rpm -ql openvswitch) /usr/bin/uuidgen /usr/bin/hostname /usr/bin/touch /usr/bin/expr /usr/bin/id /usr/bin/install /usr/bin/setpriv /usr/bin/nice /usr/bin/df
+    # By default, ovsdb-server.service runs as USRE=openvswitch, However
+    # openvswitch doesn't have the permission to write to /tmp in kdump initrd
+    # and ovsdb-service.servie will failure with the error
+    # "ovs-ctl[1190]: ovsdb-server: failed to create temporary file (Permission denied)"
+    # So run ovsdb-server.service as root instead
+    mkdir -p "${initdir}/etc/sysconfig/system.conf.d"
+    echo "root:root" >"${initdir}/etc/sysconfig/openvswitch"
+    # Bypass the error "referential integrity violation: Table Port column interfaces row"
+    # caused by changing the connection profiles
+    echo "OPTIONS=\"--ovsdb-server-options='--disable-file-column-diff'\"" >>"${initdir}/etc/sysconfig/openvswitch"
+
+    KDUMP_DROP_IN_DIR="${initdir}/etc/systemd/system/nm-initrd.service.d"
+    mkdir -p $KDUMP_DROP_IN_DIR
+    printf "[Unit]\nAfter=openvswitch.service\n" >$KDUMP_DROP_IN_DIR/01-after-ovs.conf
+
+    $SYSTEMCTL -q --root "$initdir" enable openvswitch.service
+    $SYSTEMCTL -q --root "$initdir" add-wants basic.target openvswitch.service
+}
+
```

Comment:
Hi @prudo1! Yes, it's better to move it to dracut or even this kdump dracut module in the long term. Unfortunately, this may need more code-refactoring than  we have expected. In order for our kdump dracut module tell dracut to include the future ovs dracut module, we need to move the logic of detecting if it's an OVS bdrige to either the check function or mkdumprd.

Reply:
Hi @jbtrystram! How soon do you expect [RHEL-33465](https://issues.redhat.com/browse/RHEL-33465) to be resolved? It's ideal to have an upstream ovs dracut module but it will takes much more time. So we have a short term fix firstly if that's what you prefer.

Reply:
@coiby thanks for the ping !  Having a short term fix and a tracking issue to make sure we adjust when the dracut module lands is fine with me ! 
Thanks for the work